utils/tc_marian_logs.py

```diff
@@ -1,16 +1,49 @@
 """
```

Comment:
Suggestion (optional): `tc_marian_logs.py` is not really a good name for this file now, as it's more of a generic downloader.